control-plane/subcommand/common/metrics_util.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import "strconv"
+
+func GetScrapePort(v string) (int, bool) {
```

Comment:
nit: can we rename these to be `ParseScrapePort` and `ParseScrapePath`, `Get` implies we're getting the value from somewhere (like a map, slice, or some external datasource) but really we're taking a value and parsing it